yarn.lock

```diff
@@ -1321,6 +1321,15 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@babel/runtime@npm:^7.14.8":
```

Comment:
@ibacher  @denniskigen  what is the protocol for updating yarn.lock files?  I assume this is okay?

Reply:
Looks like the `yarn.lock` changes are causing some of the CI checks to fail. I only really need to change `esm-framework` to grab the most recent `next` version. Any advice?